Source/APIClient/Request.swift

```diff
@@ -34,15 +36,22 @@ public protocol Request {
 
 public extension Request {
 
+    var contentType: String {
+        switch self {
+        default:
+            return "application/json"
+        }
+    }
+
     var urlRequest: URLRequest {
         let baseURL = Foundation.URL(string: Self.baseURL)!
-        let url = Foundation.URL(string: path, relativeTo: baseURL)!
+        let url = Foundation.URL(string: path, relativeTo: baseURL) ?? baseURL
```

Comment:
Perhaps a better option is to revert this to how it was, and then handle this on a case-by-case basis. So instead of
`let url = Foundation.URL(string: path, relativeTo: baseURL) ?? baseURL`

something like
```swift
let baseURL = Foundation.URL(string: Self.baseURL)!
let url = Foundation.URL(string: path, relativeTo: baseURL)!
// etc...
case someCaseWithoutAPath
// etc...
var path: String {
    switch self {
        case .someCaseWithoutAPath:
            return Self.baseURL
    }
}

```